tests/phpunit/tests/test-wpml-config.php

```diff
@@ -336,4 +336,47 @@ public function test_register_string_object() {
 		$this->prepare_options( 'OBJECT' );
 		$this->_test_register_string();
 	}
+
+	public function test_gutenberg_blocks() {
+		PLL_WPML_Config::instance()->init();
+
+		$expected_parsing_rules                = array(
+			'core/image'        => array(
+				'//figure/figcaption',
+				'//figure/img/@alt',
+			),
+		);
+		$expected_parsing_rules_for_attributes = array(
+			'core/navigation-link'      => array(
+				'description',
+			),
+		);
+
+		$parsing_rules                = apply_filters( 'pll_blocks_xpath_rules', array() );
+		$parsing_rules_for_attributes = apply_filters( 'pll_blocks_rules_for_attributes', array() );
+
+		$this->assertSameSets( $expected_parsing_rules, $parsing_rules, 'Rules should not be added to the existings ones' );
```

Comment:
>'Rules should not be added to the existings ones'

I don't understand.

Reply:
Done in f6e1b5ab78efce4054fe26764e3f9c8e891b99c4